cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -481,6 +481,36 @@ describe('Dashboards', () => {
         cy.getByTestID('dashboard-card').should('have.length', 1)
         cy.contains(dashboardName)
       })
+
+      it('can persist search term', () => {
+        cy.getByTestID('dashboard-card').should('have.length', 2)
+
+        cy.getByTestID('search-widget').type(dashSearchName)
+
+        cy.getByTestID('dashboard-card').should('have.length', 1)
+
+        cy.getByTestID('dashboard-card')
+          .first()
+          .getByTestID('dashboard-card--name')
+          .should('contain', dashboardName)
+
+        // Navigate Away and come back using browser's back button
+        cy.getByTestID('dashboard-card')
+          .first()
+          .getByTestID('dashboard-card--name')
+          .click()
+        cy.wait(1000)
```

Comment:
Can we handle the navigation without these hard coded wait here and below?  (And looking [here](https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting) I'm not sure if both of these are necessary.  Perhaps you need to add a cy.get after the click on 501 before you go back though? Not sure on that.)